components/metrics/DataTable.js

```diff
@@ -15,13 +16,31 @@ export default function DataTable({
   renderLabel,
   height,
   animate = true,
+  fetchMoreItems,
   virtualize = false,
+  limit = 10,
 }) {
+  const [paginatedData, setPaginatedData] = useState(data);
+  const [hasNextPage, setHasNextPage] = useState(data.length === limit);
   const [format, setFormat] = useState(true);
   const formatFunc = format ? formatLongNumber : formatNumber;
-
   const handleSetFormat = () => setFormat(state => !state);
 
+  const isItemLoaded = index => !hasNextPage || index < paginatedData.length;
+  const onLoadMoreItems = async () => {
```

Comment:
Some time later it magically works 🤷🏼‍♂️
But I think we need a little loading indicator that indicates that more data is being loaded.

Also, can the pages (especially the first) contain ~20 nodes? Otherwise it fetches multiple times right a away without the user interacting at all.

Reply:
Will check contries and update to 20 nodes.
You're right, I didn't test all the tabs yet.